Slim/Formats/Playlists/CUE.pm

```diff
@@ -337,9 +772,117 @@ sub parse {
 		return;
 	}
 
+        #dump $tracks;
 	return $tracks;
 }
+sub _addCommand{
```

Comment:
It Actually adds commands and rem comments.

Command is the proper name for real commands, strings after a REM command are more like comments, just avoid to use comment becouse is a special world for LMS in cue sheet. I used rem Command in code and the generic name of command for both, we could change using the generic name of xxx and then this will be _addXxx. Just unable to find out a valid name, since TAG and attribute are used with other meaning.